frontend/javascripts/libs/UpdatableTexture.ts

```diff
@@ -152,7 +152,6 @@ class UpdatableTexture extends THREE.Texture {
       src,
     );
     this.gl.bindTexture(this.gl.TEXTURE_2D, activeTexture);
-    this.image = null;
```

Comment:
This resource is necessary when re-uploading textures to the GPU after a context recovery. I compared the heap memory consumption between this branch and the master and could not find a noticeable difference.